linux-5.15.defconfig

```diff
@@ -123,6 +122,14 @@ CONFIG_IP_NF_ARPTABLES=m
 CONFIG_IP_NF_ARPFILTER=m
 CONFIG_IP_NF_ARP_MANGLE=m
 CONFIG_BRIDGE=m
+CONFIG_CAN=y
+CONFIG_CAN_MCP251X=m
+CONFIG_BT=y
```

Comment:
Not needed on PLT-300, BLE is handled completely in userland.
Is needed on PLT-200.